### PR TITLE
Fix relocation of files in run-instr-gen for riscv_csr_test

### PR DESCRIPTION
**Pull request by rswarbrick**
Commit a628007 broke this test (which uses different paths from all
the others: aargh!)

### CODE REVIEW

File: dv/uvm/core_ibex/scripts/run-instr-gen.py [hunk 1]
```diff
@@ -208,5 +204,34 @@ def reloc_word(simulator: str,
...
+    '''
+    for src_name, dst_name, copy_on_fail in copy_rules:
+        if run_failed and not copy_on_fail:
+            continue
+
+        src_path = os.path.join(src_dir, src_name)
+        dst_path = os.path.join(dst_dir, dst_name)
+
+        if os.path.exists(src_path):
+            print(f'SRC: {src_path}; DST {dst_path}')
```
Comment (hcallahan-lowrisc) [inline dv/uvm/core_ibex/scripts/run-instr-gen.py:225]:
```suggestion
```

Comment (hcallahan-lowrisc) [inline dv/uvm/core_ibex/scripts/run-instr-gen.py:225]:
```suggestion
```

**Review by hcallahan-lowrisc — APPROVED** [review]
Just a nit. Looks good otherwise.

File: dv/uvm/core_ibex/scripts/run-instr-gen.py (hunk 1 above)
Comment (rswarbrick) [reply dv/uvm/core_ibex/scripts/run-instr-gen.py]: Oops!

**Pull request merged**